cmd/login.go

```diff
@@ -354,14 +353,41 @@ var loginCmd = &cobra.Command{
   if are not already signed in to the account
 * You'll see "Successfully authenticated" on success
 * You can now return to the terminal and start using the CLI`,
-	Example: `tigris login api.preview.tigrisdata.cloud`,
+	Example: `
+# Login to the hosted platform
+tigris login api.preview.tigrisdata.cloud
+
+# Point all subsequent commands to locally running instance
+tigris login local
```

Comment:
Since the local Tigris instance is interacted with by using the term `dev`, can we change this to be `dev` instead of `login` ?

Reply:
`tigris login dev` you mean?

Reply:
Yup

Reply:
Done.